demo/index.html

```diff
@@ -72,47 +79,60 @@
     import bookmarksController from './bookmarks_controller.js';
 
     const bookmarks = [{
-      id: 1,
-      thumbnail: '//placehold.it/320x496',
-      page: 'xii',
-    }, {
-      id: 2,
-      thumbnail: '//placehold.it/320x496/06c/fff',
-      page: 9,
       note: 'This is a long comment I left about this bookmark in order to test out the display in the panel on the side of the bookreader.',
-    }, {
-      id: 3,
+      color: 0,
+      id: '2',
+      leafNum: '2',
+      page: '-2-',
```

Comment:
Is this `-2-` a special value?

Reply:
yes, it is the display for the page.  i am just updating the schema for the demo to reflect what we use in petabox.  Fortunately, we don't have to make the change for this value internally in this repo (bc that is coming next!)